phpstan.neon

```diff
@@ -13,6 +13,227 @@ parameters:
     stubFiles:
         - stubs/glpi_constants.php
     ignoreErrors:
+        - '/Access to an undefined property CommonDBTM::\$orderby\./'
```

Comment:
Most of ignored errors have to be fixed. FOal is not to have today a level2 compliant code, but to fix some issues so we can reach this target.

Reply:
The aim isn't to keep these ignore rules, it's just temporary while the PR is being drafted, to help me keep track of progress and test with the CI.

The 1st commit lists all the errors found. Then in subsequent commits, I remove the fixed rules, as in the 2ᵉ commit “PHPDoc”.